vignettes/chapter7_DescribingDataWithoutKnowingSleep.Rmd

```diff
@@ -33,59 +33,59 @@ Average acceleration is known to be correlated with the activity-related energy
+To get a more detailed description of the data, looking at the distribution of acceleration values can be informative. GGIR facilitates this in two ways:
 
-1.  By specifying the quantiles of the distribution with parameter `qlevels`, which are fed to build-in R function `quantile` and give us the metric values corresponding to such quantiles (a quantile multiple by 100 is the same as a percentile).
+1.  By specifying the quantiles of the distribution with parameter `qlevels`, which are fed to the build-in R function `quantile`, GGIR gives us the metric values corresponding to such quantiles (a quantile multiplied by 100 is the same as a percentile).
 
-2.  By describing time spent in acceleration ranges, which are defined by parameter `ilevels` .
+2.  By describing the time spent in acceleration ranges, which are defined by parameter `ilevels` .
 
-In the physical activity literature the distribution of acceleration values is often referred to as intensity distribution.
+The distribution of acceleration values is often referred to as intensity distribution in the physical activity literature.
 
 ## Derived descriptives
 
 ### Sets of quantiles (MX metrics by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/29360664/))
 
-The quantiles as discussed above can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
-In the specific approach as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/) these quantiles are referred to as the **MX metrics**.
+The quantile,s as discussed above, can be used to describe the accelerations that participants exceed in their most active “X” accumulated minutes in a day.
+In the specific approach, as proposed by [Rowlands et al.](https://pubmed.ncbi.nlm.nih.gov/31808014/), these quantiles are referred to as the **MX metrics**.
 The MX metrics should not be confused with the most active continuous X hours, e.g.
 M10, as used in circadian rhythm research that also can be derived with GGIR (see parameter `winhr`).
 
-To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day that you wish to filter out the accelerations for.
-For example, to generate the minimum acceleration value for the most active 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
-This can also be used as a nested term to generate multiple metrics, for example to call M60, M30 and M10, you can use the following parameter:
+To use the MX metrics as proposed by Rowlands et al., specify the durations of the 24h day during which you want to filter out the accelerations values.
+For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day.
```

Comment:
Please rephrase:
"For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will filter out the lowest 1410 minutes of the day."

to

"For example, to generate the minimum acceleration value for the most active accumulated 30 minutes, you can call `qlevels = (1410/1440)`, which will identify the acceleration corresponding to the most active 30 minutes in a 1440 minute day."

Reply:
Agree, revise this accordingly.